Integrations/integration-opswat-metadefender.yml

```diff
@@ -19,7 +19,7 @@ configuration:
   defaultvalue: ""
   type: 4
   required: false
-- display: Trust any certificate (unsecure)
```

Comment:
Why did you change it? Should be as it was

Reply:
Because it should be: "Do not validate server certificate (insecure)"

Reply:
No, we changed it.

Reply:
Changed